website/docs/artist_tools_context_manager.md

```diff
@@ -0,0 +1,17 @@
+---
+id: artist_tools_context_manager
+title: Context Manager
+sidebar_label: Context Manager
+description: A tool to manage the context within a host app.
+---
+
+# Context Manager
```

Comment:
Context manager was actually merged with Workfiles tool so Context manager does not exist anymore.

Reply:
Oops, anyhow this shouldn't be in this PR, will fix and add you to the other PR